backend/pkg/database/database.go

```diff
@@ -122,19 +139,29 @@ SELECT applications.*,
        aps.total_high_count,
        aps.total_critical_count,
        aps.highest_severity,
-       aps.lowest_severity
```

Comment:
why this is gone?

Reply:
by mistake, will add it back but we don't really need that value.